vllm/model_executor/openvino_model_loader.py

```diff
@@ -128,8 +130,8 @@ def callback(m: Matcher) -> bool:
                 real_v = mapping[v_current]
                 hidden_shape = real_q.get_partial_shape()
                 hidden_dim = hidden_shape[hidden_shape.rank.get_length() - 1].get_length()  # TODO: What if it is a dynamic? Need to insert a ShapeOf sub-graph instead
-                k_parameter = opset13.parameter(shape=[-1, -1, -1, -1, -1], dtype=np.float32)
-                v_parameter = opset13.parameter(shape=[-1, -1, -1, -1], dtype=np.float32)
+                k_parameter = opset13.parameter(shape=[-1, -1, -1, -1, -1], dtype=kv_cache_dtype)
+                v_parameter = opset13.parameter(shape=[-1, -1, -1, -1], dtype=kv_cache_dtype)
```

Comment:
Is it correct expectation, that CPU doesn't require any specific dimension in this part except the correct data type? But for GPU, specific dimensions is strongly required in static form?

Reply:
> But for GPU, specific dimensions is strongly required in static form?

Why? I expect GPU can also work with dynamic shapes